pkg/pull/pull.go

```diff
@@ -292,6 +292,10 @@ func Pull(upstreamURI string, pullOptions PullOptions) (string, error) {
 		}
 
 		for _, prevChart := range prevHelmCharts {
+			if prevChart.Spec.Exclude.String() == "1" {
```

Comment:
What is prevChart here? The currently deployed version or the last rendered version?

Reply:
I believe it's the last rendered version, but I am not sure right now - @divolgin ?

Reply:
it's the previous rendered version, `sequence - 1`

Reply:
Seems like it'd be better to compare to the actually deployed version, but I don't know how hard that is to do.